From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@rmca14__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-romcal). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Do not approve.

Commit:
drafting

Commit:
drafting

Commit:
drafting

Commit:
drafting

Commit:
drafting

Commit:
removing load balancer

Commit:
drafting

Commit:
removing load balancer

Commit:
fixing script

Commit:
safekeeping

Commit:
safekeeping

Commit:
safekeeping

Commit:
links and art

Commit:
storage account changes

Commit:
fixing links

Commit:
acrolinx

Commit:
fixing links

Commit:
moving the furniture

Commit:
testing

Commit:
hurmph

Commit:
adding in link to app install script

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into iis-sql

Commit:
new repo

Commit:
clean-up and test

Commit:
Merge branch 'master' of https:/github.com/MicrosoftDocs/azure-docs-pr into iis-sql

Commit:
TOC

Commit:
fixing link from previous tutorial

Commit:
Merge branch 'master' of https://t push origin iis-sq	github.com/MicrosoftDocs/azure-docs-pr into iis-sql

Commit:
Iains comments

Commit:
reverting from 2.0

Commit:
adding back in the firewall rule

rjagiewich committed:
edit pass: azure-stack-delegated-provider

rjagiewich committed:
edit pass: azure-stack-delegated-provider

rjagiewich committed:
edit pass: azure-stack-delegated-provider

rjagiewich committed:
edit pass: azure-stack-delegated-provider

Commit:
moving to new-azvm and SQL VM

Commit:
vnet stuff

Commit:
subnet stuff

mahesh-unnikrishnan committed:
New article on comparison between AAD-DS and Azure AD Join

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
extension

Commit:
finializing

Commit:
merging

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into iis-sql

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

mahesh-unnikrishnan committed:
Update active-directory-ds-compare-with-azure-ad-join.md

mahesh-unnikrishnan committed:
Add an entry to TOC.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr; branch 'master' of https://github.com/Microsoft/azure-docs-pr

mahesh-unnikrishnan committed:
Update active-directory-ds-compare-with-azure-ad-join.md

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr

Commit:
Dans feedback

Commit:
iot-hub-node-node-file-upload tutorial

Commit:
TOC and selector additions

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
fixing link from CI\CD

Commit:
fixing links

v-nagta committed:
USER STORY 1107498  SaaS App Tutorial: Grovo

v-nagta committed:
updated

v-nagta committed:
updated

divyaswarnkar committed:
soap1

divyaswarnkar committed:
soap2

divyaswarnkar committed:
soap3

divyaswarnkar committed:
soap4

Commit:
naming

Commit:
acrolinx

Commit:
adding slash

Commit:
backslash

Commit:
adding sizes back to the TOC

v-nagta committed:
updated

divyaswarnkar committed:
soap5

Commit:
fixing linux double toc thing

Commit:
initial revision per comments

divyaswarnkar committed:
soap6

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr

mahesh-unnikrishnan committed:
Update screenshots for the getting started article to remove reference to
classic vnet.

mahesh-unnikrishnan committed:
Update screenshot to remove reference to classic vnets.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr

mahesh-unnikrishnan committed:
Minor cleanup & inline recommendations.

mahesh-unnikrishnan committed:
New article on disabling AAD-DS.

Commit:
adding link to landing page per Kris

mahesh-unnikrishnan committed:
Update TOC.md

mahesh-unnikrishnan committed:
Update TOC.md

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mahesh-unnikrishnan committed:
Add border to screenshots.

mahesh-unnikrishnan committed:
acrolinx fixes

mahesh-unnikrishnan committed:
layout fixes

v-nagta committed:
updated

divyaswarnkar committed:
soap7

mahesh-unnikrishnan committed:
Add video on AAD-DS overview

mahesh-unnikrishnan committed:
Updates to comparison doc.

mahesh-unnikrishnan committed:
Fix embed link

mahesh-unnikrishnan committed:
acrolinx fixes

mahesh-unnikrishnan committed:
more acrolinx fixes

mahesh-unnikrishnan committed:
layout cleanup

mahesh-unnikrishnan committed:
Fix table layout bug

Commit:
second revision per review

Commit:
third revision per second wave of comments

Commit:
removed unneeded links, images and fixed typo

Commit:
fixed image path references

Commit:
Update index.md

mahesh-unnikrishnan committed:
Update active-directory-ds-disable-aadds.md

gopitk committed:
SQL 2017 and MLS update

gopitk committed:
SQL 2017 and Ml Server updates

gopitk committed:
New screenshot

gopitk committed:
More SQL 2017 and ML Server changes

gopitk committed:
Minor typo and grammar

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

TerryLanfear committed:
final updates

v-nagta committed:
updated

cephalin committed:
add backup PS scripts

cephalin committed:
acrolinx

TerryLanfear committed:
final update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

TerryLanfear committed:
update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into septrel_01

rayne-wiselman committed:
redirecting two unused files

Commit:
added indentation and condensed function per latest comments

ShawnJackson committed:
Update azure-stack-delegated-provider.md

spelluru committed:
added link to SDK reference content

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into monitorsdks1030

spelluru committed:
formatting fix

spelluru committed:
fixed spelling

HeidiSteen committed:
comparison table

Lisaco88 committed:
Merge pull request #26462 from TerryLanfear/septrel_01

Septrel 01

spelluru committed:
minor fix

ShawnJackson committed:
Merge pull request #25313 from rjagiewich/azure-stack-delegated-provider

Edit pass: Azure stack delegated provider

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits to the table

Commit:
Merge pull request #26487 from spelluru/execpipeline1030

formatting fix

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

PRMerger14 committed:
Merge pull request #26489 from spelluru/inga1030

minor fix

PRMerger17 committed:
Merge pull request #26486 from spelluru/monitorsdks1030

added link to SDK reference content

YuriDio committed:
updating article

douglaslMS committed:
Removed links to PDF per Craig's suggestion.

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

Lisaco88 committed:
Merge pull request #26404 from dend/patch-2

Links should never include direct monikers

HeidiSteen committed:
minor edits

GitHubber17 committed:
Merge pull request #26062 from v-nagta/grovo

USER STORY 1107498  SaaS App Tutorial: Grovo

douglaslMS committed:
Changed ownership to PMs in metadata of 4 Sync topics.

PRMerger17 committed:
Merge pull request #26497 from YuriDio/OctoberUpdates_17

updating article

HeidiSteen committed:
more edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

spelluru committed:
minor fix

mimig1 committed:
Cosmos DB: Added step to save file.

ktoliver committed:
Merge pull request #26245 from mahesh-unnikrishnan/master

Docs updates for AAD-DS

v-thepet committed:
today's broken links

CarlRabeler committed:
a-d

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
edits

GitHubber17 committed:
Merge pull request #26176 from cynthn/toc-sizes

Adding sizes back to the TOC

CarlRabeler committed:
i-t

CarlRabeler committed:
t-z

BryanLa committed:
clarity on SSH key config

brapel committed:
custom-search: minor tweaks

PRMerger17 committed:
Merge pull request #26501 from v-thepet/fixy

production bug: today's broken links

v-shils committed:
Merge pull request #26459 from gopitk/patch-7

Updates with support for MLS and SQL 2017

PRMerger15 committed:
Merge pull request #26500 from mimig1/patch-149

Cosmos DB: Added step to save file.

CarlRabeler committed:
acro

Commit:
removed images

douglaslMS committed:
Changed owner of 2 main topics back to me for now.

Commit:
Merge pull request #26508 from brapel/custom-search-small-tweaks

custom-search: minor tweaks

carolinacmoravia committed:
M13816: Possible typo mistake: "models"

 Hi, @yzhong94,

Translator has reported possible source content issue. "hardware security models" seems to be a typo of "hardware security modules" because "HSM" is an acronym of "Hardware Security Module." In addition, "Hardware Security Modules" are used in the subsequent segments.

Please, check the proposed file change and help to merge if agreed. 

Many thanks in advance.

v-shils committed:
Merge pull request #26097 from divyaswarnkar/soap

Soap

Commit:
Add NOTE about Preview status. Discussed with Amol Agarwal. MightyPen = genemi.

Commit:
Merge pull request #26499 from douglaslMS/sync-del-pdf

Removed links to PDF per Craig's suggestion.

PRMerger14 committed:
Merge pull request #26493 from HeidiSteen/heidist-master

comparison table

Commit:
Eliminate blank line before bullet, in new NOTE.

Ja-Dunn committed:
Merge pull request #26511 from swhite-msft/master

removed images

Ja-Dunn committed:
Merge pull request #26505 from CarlRabeler/workload-sql-3

i-t

carolinacmoravia committed:
M13818: Possible typo error: "Ongoing"

Hello, @YuriDio,

Translator has reported possible source content issue. "Ongoing" is a typo of "Onboarding" because this page has a section "Onboarding non-Azure resources."

Please, check the proposed file change and help to merge if agreed.

Thank you.

Ja-Dunn committed:
Merge pull request #26502 from CarlRabeler/workload-sql-1

a-d

Ja-Dunn committed:
Merge pull request #26483 from rayne-wiselman/rayne-october30-2

redirecting two unused files

LizCasey committed:
Pull request to update author field in order to retire various articles.

Commit:
sanitized example

v-shils committed:
Merge pull request #26040 from msebolt/iot-hub-node-node-file-upload

Iot hub node node file upload

PRMerger12 committed:
Merge pull request #26514 from MightyPen/vnetregions-20171030-a-4sb2

Add NOTE about Preview status.

CarlRabeler committed:
acro

Commit:
Merge pull request #26518 from carolinacmoravia/patch-417

M13818: Possible typo error: "Ongoing"

HeidiSteen committed:
bing api and links

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

Ja-Dunn committed:
Merge pull request #26520 from swhite-msft/scottwhi-hit-highlighting

sanitized example

Commit:
Tweaking PR 26514. Removing mention of PowerShell. Per Amol.A.  MightyPen = genemi.

PRMerger19 committed:
Merge pull request #26522 from LizCasey/v-elicas1027

Pull request to update author field in order to retire various articles.

PRMerger12 committed:
Merge pull request #26524 from HeidiSteen/heidist-master

Heidist master

Ja-Dunn committed:
Merge pull request #25619 from cynthn/iis-sql

IIS\SQL Stack Tutorial

Commit:
2. Using Amol's phrasing.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

CarlRabeler committed:
Merge branch 'master' into workload-sql-4

PRMerger12 committed:
Merge pull request #26513 from carolinacmoravia/patch-416

M13816: Possible typo mistake: "models"

v-alje committed:
Merge pull request #26531 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Minor additions.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasgw

Ja-Dunn committed:
Merge pull request #26470 from cephalin/powershell

add backup PS scripts

Commit:
Merge pull request #26527 from MightyPen/vnetregions-20171030-a-4sb2

Tweaking PR 26514, re VNet for SQL DB.

PRMerger12 committed:
Merge pull request #26507 from BryanLa/bryanla-msi-sshhelp

clarity on SSH key config

Commit:
Merge pull request #26528 from spelluru/inga1030-2

minor fix

Ja-Dunn committed:
Merge pull request #26506 from CarlRabeler/workload-sql-4

t-z

Commit:
Merge pull request #26535 from Minewiskan/aasgw

Aasgw
